AdvLoggerPkg/Application/DecodeUefiLog/DecodeUefiLog.py

```diff
@@ -293,27 +293,27 @@ class AdvLogParser ():
     # #define DEBUG_ERROR  0x80000000          // Error
 
     debug_levels_dict = {
-        0x00000001: "[DEBUG_INIT]",

```

Comment:
Do the strings really need `DEBUG_`? For example, can this just be `[INIT]`?

Reply:
`DEBUG_` is required as the words like `init` or `error` are very commonly used words in the debug print messages, so it will cause lots of noise if we only use the `error` itself.

Reply:
I understand `init` and `error` are common, but is `[INIT]`? This seems unnecessarily verbose as a log marker that is going out over interfaces like serial for every message. It will also have a substantial impact on overall log file sizes and memory buffer usage.

Reply:
That's a good point, for certain key words like `[ERROR]` we do have some usage already in the tree, but they are at the right place indicating the right debug level so we are good.I have tested and updated the relevant part in both files. Thanks Michael.